src/steps/malwares/converters.ts

```diff
@@ -0,0 +1,24 @@
+import { createIntegrationEntity } from '@jupiterone/integration-sdk-core';
+import { Malware } from '../../types';
+import { Entities } from '../constants';
+
+function getMalwareKey(id: string): string {
+  return `malware:${id}`;
+}
+
+export function createMalwareEntity(data: Malware) {
+  return createIntegrationEntity({
+    entityData: {
+      source: data,
+      assign: {
+        _class: Entities.MALWARE._class,
+        _type: Entities.MALWARE._type,
+        _key: getMalwareKey(data.guid),
+        category: 'malware',
+        open: data.status === 'Detected' ? true : false,
+        severity: data.score.toString(), // doubtful
+        numericSeverity: data.score,
```

Comment:
The API only provides the severity in terms of numerics (in the form of the field score). Not sure how we should handle converting them to 'strings'.

Reply:
Let's tackle this once we're done with "big pieces" first. We could try of thinking about some conversion, but if we fail to come up with a solution we think make perfect sense, we can always ask JupiterOne people as well!